### PR TITLE
Fabo/lint staged

### PR DESCRIPTION
**Pull request by faboweb**
Only lint staged files and also apply style linting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,7 +129,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "yarn lint --no-fix"
+      "pre-push": "lint-staged"
```
Comment (Bitcoinera) [inline package.json:132]: Shouldn't `lint-staged` be a script?

Comment (Bitcoinera) [reply package.json:132]: Oh, so by doing `npx mrm lint-staged` then it should be fine right?

But do we all need to do this in our machines in order for it to work?

Comment (faboweb) [reply package.json:132]: the package is already installed: https://github.com/luniehq/lunie/pull/3841/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R114

Comment (Bitcoinera) [reply package.json:132]: Did you try this? Making a push with this?

Comment (faboweb) [reply package.json:132]: yep, works. to test just fork, change a file (disabling vscode auto format before) and push

Comment (Bitcoinera) [reply package.json:132]: It's fine, I believe you 😉

**Comment by Bitcoinera** [comment]
Just missing changelog!

**Comment by faboweb** [comment]
image

**Comment by Bitcoinera** [comment]
> Weird but ur right

Should E2E also test if there is a changelog included?

**Pull request merged**